File: src/main/java/com/cuupa/classificator/services/kb/semantic/Metadata.java
package com.cuupa.classificator.services.kb.semantic;

public class Metadata {

	private final String name;
	private final String value;

	public Metadata(String name, String value) {
		this.name = name;
		this.value = value;
	}

	public String getName() {
		return name;
	}

	public String getValue() {
		return value;
	}
}

File: src/main/java/com/cuupa/classificator/knowledgebase/resultobjects/Metadata.kt
package com.cuupa.classificator.knowledgebase.resultobjects

import com.cuupa.classificator.knowledgebase.services.token.Token

data class Metadata(val name: String, val value: String) : SemanticResultData() {
    override fun addToken(token: Token) {
        //noting to do here
    }
}